examples/openshift-origin/README.md

```diff
@@ -2,12 +2,11 @@
 
 This example shows how to run OpenShift Origin as a pod on an existing Kubernetes cluster.
 
-OpenShift Origin runs with a rich set of role based policy rules out of the box that requires authentication from users
-via certificates.  When run as a pod on an existing Kubernetes cluster, it proxies access to the underlying Kubernetes services
-to provide security.
+OpenShift Origin runs with a rich set of role based policy rules out of the box that requires authentication from users via certificates.  When run as a pod on an existing Kubernetes cluster, it proxies access to the underlying Kubernetes services to provide security.
 
-As a result, this example is a complex end-to-end configuration that shows how to configure certificates for a service that runs
-on Kubernetes, and requires a number of configuration files to be injected dynamically via a secret volume to the pod.
+As a result, this example is a complex end-to-end configuration that shows how to configure certificates for a service that runs on Kubernetes, and requires a number of configuration files to be injected dynamically via a secret volume to the pod.
+
+This example will create a pod running the OpenShift Origin master as well as three pods running a stand-alone etcd service, and another pod used for the etcd service discovery and initialization.
```

Comment:
It would be good to elaborate here some more.

"This example will create a pod running the OpenShift Origin master.  In addition, it will run a three-pod etcd setup to hold OpenShift content.  When OpenShift runs on top of an existing Kubernetes cluster, content specific to Kubernetes will be stored in the Kubernetes etcd repository (i.e. pods, services, replication controllers, etc.), but OpenShift specific content (builds, images, users, policies, etc.) are stored in its etcd setup".